src/WPF/WPF.Viewer/SettingsWindow.xaml.cs

```diff
@@ -68,9 +69,12 @@ public SettingsWindow()
 
             SetUpTelemetryTab();
 
-#if STORE_RELEASE
+#if RELEASE
             ScreenshotTab.Visibility = Visibility.Collapsed;
-            APIKeyTab.Visibility = Visibility.Collapsed;
+            if(ApiKeyManager.DisableApiKeyUI)
+            {
+                APIKeyTab.Visibility = Visibility.Collapsed;
+            }
```

Comment:
I'd move this outside "RELEASE" and just always be a thing if it's set.

Reply:
Done.